topic-operator/src/main/java/io/strimzi/operator/topic/v2/BatchingTopicController.java

```diff
@@ -1020,4 +1042,16 @@ private static boolean isDifferentCondition(Condition oldReadyCondition,
                 || !Objects.equals(oldReadyCondition.getReason(), newReason)
                 || !Objects.equals(oldReadyCondition.getMessage(), newMessage);
     }
+
+    private void startTimer(ReconcilableTopic topic) {
+        if (topic.timerSample() == null) {
+            topic.timerSample(Timer.start(metrics.metricsProvider().meterRegistry()));
```

Comment:
These names should include what we're timing. E.g. `startReconciliationTimer()` and `topic.reconciliationTimerStample()`. Better convey (when reading the code) what it is that we're timing, and allow us to add other timers later on.